validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -474,6 +488,23 @@ def stop(self):
                 # Connection has already been disconnected.
                 pass
 
+    def _request_chain_head(self):
+        peers = self._gossip.get_peers()
+
+        peered_connecions = [conn_id for conn_id in peers
+                             if self._connection_statuses[conn_id] ==
+                             PeerStatus.PEER]
+
+        if not peered_connecions:
+            return
+
+        LOGGER.debug('Have not received a chain head from peers. '
+                     'Requesting from %s',
+                     peered_connecions)
```

Comment:
Per @nick-drozd logging guidelines, "Logging should not be done by low-level functions" this log message should be removed or moved to a higher level function.

Reply:
This is potentially the only place to detect this change, so the log message lives at its appropriate level in the code.

Reply:
I was suggesting moving it to the `run()` method which is high-level instead of in this lower level helper function. Line 471 in this file